docs/blog/october-contributions-showcase.md

```diff
@@ -0,0 +1,107 @@
+---
+
+# September contributions
+
+_October 4, 2024_
+
+A few days late again because I was caught in another typhoon :grimacing:, but here's the latest wins from the e18e community!
+
+# Libraries
+
+## picospinner & nanospinner
+
+Two small but great additions, these CLI spinner libraries are here to replace heavier weight alternatives like `ora`.
+
+[`picospinner`](https://github.com/tinylibs/picospinner/) was recently adopted into the [tinylibs](https://github.com/tinylibs) organisation, and is being lead by [Pondwader](https://github.com/PondWader).
+
+[`nanospinner`](https://github.com/usmanyunusov/nanospinner) author [@usmanyunusov](https://x.com/usmanyunusov) has also recently joined the e18e community and is actively helping move things forward.
+
+Both of these libraries are great alternatives and unblock a whole bunch of future migrations. Looking forward to see more!
+
+## empathic / fd-package-json
+
+As part of the cleanup project, we needed an alternative package for finding the closest `package.json`. To fill the gap, [`fd-package-json`](https://github.com/es-tooling/fd-package-json) was created **and recently passed 800,000 downloads/week!**
+
+However, we still had many gaps where we wanted to traverse for other files and directories without having to pull in a heavy library.
+
+This is where [empathic](https://github.com/lukeed/empathic) comes in!
+
+Another great library by [@lukeed](https://x.com/lukeed05), this provides a bunch of useful utilities for dealing with paths and traversals.
+
+This library can easily replace many of the `find-up`-like packages floating around the ecosystem:
+
+```ts
+import * as find from 'empathic/find'
+
+// Find closest "foobar.config.js" file
+const file = find.up('foobar.config.js', { cwd })
+```
+
+We will soon be migrating many projects to both of these libraries for sure :raised_hands:
+
+# Contributions / Improvements
+
+## Storybook
+
+## Docusaurus
+
+This one didn't originate from e18e but is still well worth a mention. [Docusaurus](https://github.com/facebook/docusaurus) is currently tackling a pretty huge project of [improving build performance](https://github.com/facebook/docusaurus/issues/10556).
+
+Great to see another large project taking a focus on performance :pray:
+
+They have already shaved off minutes from their build times :fire: very impressive work by the team.
+
+# Other news
+
+## `require(esm)` in Node
+
+[@joyeecheung](https://x.com/joyeecheung) and others have recently been working hard on allowing Node to `require(...)` ES modules.
+
+For example, this means the following code would now work:
+
+```ts
+// chai 5.x is ESM only
+const { expect } = require('chai')
+```
+
+Until now, many of us have been publishing "dual packages" - packages which contain both CommonJS and ES module syntax.
+
+This is unfortunate since it means we're often shipping double our package size in order to support both systems.
+
```

Comment:
By the way we are kicking off an initiative to better document the shipping practices and provide guides/troubleshooting for the migration:  https://github.com/nodejs/package-examples/issues/2 - kind of procrastinating it a bit until the collaboration summit session https://github.com/openjs-foundation/summit/issues/422, which would happen on Nov 8, 10:45 Dublin time, not sure remote participation is possible yet I hope it is! Though the session is mostly for awareness and brainstorming and I am sure we will get back to GitHub after it.

Reply:
would be super nice to join the summit if you can join remotely after all. I'd like to keep the community aligned with all this stuff, feels like we can do a lot here if we get it tasked out properly 🙏